interviewing/readme.md

```diff
@@ -0,0 +1,66 @@
+    - Past: Mention your relevant experience, skills, or education. Focus on achievements and how they shaped your current professional identity.
+    - Future: Highlight what you want to do next and why you’re excited about this opportunity.
+- Try to be enthusiastic, let your personality and confidence shine
+- Do not overwhelm your interviewer with too much information
+- Allow 1-2 minutes for follow-up questions
+
+After the 5 minutes are up take turns practicing your introductions. While the other person is introducing themselves, put yourself in an interviewers shoes. Share what the other person did well and what they could improve.
+
+## Step 2: The Soft Skills Interview 🥎 
+
+Soft skill interview questions are typically asked at some point during the interview process. 
+A common technique to answer these questions is to use the [STAR method](https://ca.indeed.com/career-advice/interviewing/star-interview-method) which stands for Situation, Task, Action and Result.
+
+“Can you tell me about a time when you had to work in a team and faced a challenge?”
+
+- <strong> Situation </strong>: "During my final group project, we had to develop a mobile app for managing student assignments. Midway through the project, we hit a roadblock because two teammates disagreed on which framework to use—one preferred React Native, while the other wanted to stick with Flutter. This was causing delays in our progress.”
+- <strong> Task </strong>: “As the team lead, my task was to resolve the conflict and ensure we stayed on schedule.”
+- <strong> Action </strong>: “I organized a meeting to discuss the pros and cons of both frameworks, focusing on performance and our project requirements. After evaluating, I suggested using React Native because it allowed us to quickly integrate with the backend we’d already built. The teammate supporting Flutter agreed once we reviewed the long-term benefits.”
+- <strong> Result </strong>: “We moved forward with React Native, completed the app on time, and received positive feedback from our professor for both functionality and performance. It also helped the team work more smoothly on future features.”
+
+Discuss how you might answer the following questions in a real interview.
+
+- Tell us about a time you missed a deadline.
+- What was your favorite project you have worked on so far.
+- When was a time you failed.
+- Tell us about a time you had to help your team members through a tough time. 
+- Tell me about a time you prioritised your time well, despite having multiple deadlines.
+- Tell me about a time you solved a problem at work.
+- How do you teach your colleagues about new practices within the company?
+- Tell me about a time when things didn't go according to plan. How did you deal with it?
+- What is your response when a colleague disagrees with you?
+- Share a time you set a goal and achieved it.
+
+## Step 3: The Technical Interview 📞
```

Comment:
How long should they spend asking each other? Who will say when the time is up?

Timing sessions is super important or the day descends into chaos.

Reply:
The facilitator should keep track of time and say when the participants should move on to the next section. I have added times to each stage